plugins/coreboot/fu-coreboot-common.c

```diff
@@ -23,8 +23,20 @@ fu_plugin_coreboot_version_string_to_triplet (const gchar *coreboot_version,
 	guint cb_build = 0;
 	gint rc;
 
-	rc = sscanf (coreboot_version, "CBET%u %u.%u-%u", &cb_version, &cb_major,
-			&cb_minor, &cb_build);
+	if (g_str_has_prefix (coreboot_version, "CBET")) {
+		/*
+		 * The thinkpad_acpi kernel module requires a specifc pattern
+		 * in the DMI version string. To satisfy those requirements
+		 * coreboot adds the CBETxxxx prefix to the DMI version string
+		 * on all Lenovo devices. The prefix isn't present in the
+		 * version string found in coreboot tables, or on other
+		 * coreboot enabled devices.
+		 */
+		rc = sscanf (coreboot_version, "CBET%u %u.%u-%u", &cb_version, &cb_major,
+			     &cb_minor, &cb_build);
```

Comment:
could we just do:

    if (g_string_has_prefix (coreboot_version, "CBET"))
        coreboot_version += 4

?

Reply:
That seems like a sane approach too

Reply:
done, I skipped 9 chars as the prefix is 8 + 1 whitespace